app/views/partials/personal_key/_key.slim

```diff
@@ -7,7 +7,7 @@
     = t('users.recovery_code.header')
   .navy.monospace
     - code.split(' ').each do |word|
-      p.bold.mb0.h3 data-recovery='word'
+      p.bold.mb0.h3(data-recovery="#{word}")
```

Comment:
element.innerText causes a reflow in the browser, whereas attributes don't. I guess technically in this case it doesn't matter, since the app is so light on JS, but its good practice overall.